feature-search/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I think we should use a general rule to ignore the following patter `**/build/**`